core/bower.json

```diff
@@ -37,6 +37,7 @@
     "jquery-throttle-debounce": "v1.1",
     "angular-ui-select": "0.16.1",
     "vis": "4.16.1",
-    "vivagraphjs": "latest"
+    "vivagraphjs": "latest",
+    "ansi_up": "1.3.0"
```

Comment:
hi, we already have ansi_up in dependencies https://github.com/twosigma/beaker-notebook/blob/master/core/bower.json#L33
it is also included in index_template.html https://github.com/twosigma/beaker-notebook/blob/master/core/src/main/web/app/template/index_template.html#L109